docker-hub/api/latest.yaml

```diff
@@ -0,0 +1,528 @@
+
+            Defaults to 1 month before the current time
+          type: string
+      responses:
+        200:
+          description: Success
+          schema: 
+            $ref: '#/definitions/GetNamespaceRepositoryImagesSummaryResponse'
+        401:
+          description: Unauthorized - user does not have read access to the namespace
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /v2/namespaces/{namespace}/repositories/{repository}/images:
+    get:
+      tags: [Images]
+      summary: Get details of repository's images
+      description: |
+        Limited to users on Pro and Team plans
+
+        Gets details on the images in a repository
+      operationId: GetNamespacesRepositoriesImages
+      produces:
+        - application/json
+      parameters:
+        - name: namespace
+          in: path
+          required: true
+          description: Namespace of repository
+          type: string
+        - name: repository
+          in: path
+          required: true
+          description: Name of repository
+          type: string
+        - name: status
+          in: query
+          required: false
+          description: Filters to only show images of this status
+          type: string
+          enum: [active, inactive]
+        - name: ordering
+          in: query
+          required: false
+          description: |
+            Orders the results by this property
+
+            Prefixing with `-` sorts by descending order
+          type: string
+          enum: [last_activity, -last_activity, digest, -digest]
+        - name: active_from
+          in: query
+          required: false
+          description: | 
+            Sets the time from which an image must have been pushed or pulled to be counted as active
+
+            Defaults to 1 month before the current time
+          type: string
+        - name: page
+          in: query
+          required: false
+          description: Page number to get. Defaults to 1
+          type: integer
+        - name: page_size
+          in: query
+          required: false
+          description: Number of images to get per page. Defaults to 10. Max of 100.
+          type: integer
+      responses:
+        200:
+          description: Success
+          schema: 
+            $ref: '#/definitions/GetNamespaceRepositoryImagesResponse'
+        401:
+          description: Unauthorized - user does not have read access to the namespace
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+        403:
+          description: Forbidden - this API is only available to users on Pro or Team plans
+          schema:
+            $ref: '#/definitions/ErrorResponse'
+  /v2/namespaces/{namespace}/delete-images:
+    post:
+      tags: [Images]
+      summary: Delete images
+      operationId: PostNamespacesDeleteImages
+      description: |
+        Limited to users on Pro and Team plans
```

Comment:
Same than previous comment (line 425) - deferring to @usha-mandya

Reply:
Done :)